internal/cmd/local/command_kafka_start.go

```diff
@@ -25,20 +24,28 @@ import (
 )
 
 var (
-	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Downloading", "Download complete", "Verifying Checksum", "Extracting", "Pull complete"}
+	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Download complete", "Verifying Checksum", "Pull complete"}
```

Comment:
I feel like the user would be interested in seeing all of these? (This is what they're used to seeing when they use Docker)